best_practices/creating-the-project.rst

```diff
@@ -110,6 +110,13 @@ Symfony documentation uses the AppBundle name.
     There is no need to prefix the AppBundle with your own vendor (e.g.
     AcmeAppBundle), because this application bundle is never going to be
     shared.
+    
+.. note::
+    
+    Another reason to create a new bundle is when you're overriding something 
+    in a vendor's bundle. For instance, if you have to override an action of 
+    the FOSUserBundle, then you should create a UserBundle that would be used 
```

Comment:
I took out the example here in sha: c24862be171313603f590cdccb9d8e15b4099aac, but we still have it in the linked doc. I don't think it's a huge deal, but if we can think of an easy other example, that'd be cool.

And btw - this is an awesome problem to have :+1:. I *hate* overriding bundle controllers - it's almost always a problem when upgrading. So, I like the FOSU events a lot